src/Curl/Curl.php

```diff
@@ -801,9 +844,9 @@ public function setHeader($key, $value)
      * @access public
      * @param  $function
      */
-    public function setJsonDecoder($function)
+    public function setJsonDecoder($function=null)
```

Comment:
I am not sure how I feel about the default of `null`, I would think requiring a developer to call it in full `$curl->setJsonDecoder(null);` (set it to `null`) would make more semantic sense that allowing `$curl->setJsonDecoder();` (set it to...  what?).

Just personal preference, however.